Phoenix/hello/lib/hello_web/controllers/comment_controller.ex

```diff
@@ -0,0 +1,62 @@
+defmodule HelloWeb.CommentController do
+  use HelloWeb, :controller
+
+  alias Hello.Blog
+  alias Hello.Blog.Comment
+
+  def index(conn, _params) do
+    comments = Blog.list_comments()
+    render(conn, "index.html", comments: comments)
+  end
+
+  def new(conn, _params) do
+    changeset = Blog.change_comment(%Comment{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
+  def create(conn, %{"comment" => comment_params}) do
+    case Blog.create_comment(comment_params) do
+      {:ok, comment} ->
+        conn
+        |> put_flash(:info, "Comment created successfully.")
+        |> redirect(to: Routes.comment_path(conn, :show, comment))
+
+      {:error, %Ecto.Changeset{} = changeset} ->
+        render(conn, "new.html", changeset: changeset)
+    end
+  end
+
+  def show(conn, %{"id" => id}) do
+    comment = Blog.get_comment!(id)
+    render(conn, "show.html", comment: comment)
+  end
+
+  def edit(conn, %{"id" => id}) do
+    comment = Blog.get_comment!(id)
+    changeset = Blog.change_comment(comment)
+    render(conn, "edit.html", comment: comment, changeset: changeset)
+  end
+
+  def update(conn, %{"id" => id, "comment" => comment_params}) do
+    comment = Blog.get_comment!(id)
+
+    case Blog.update_comment(comment, comment_params) do
+      {:ok, comment} ->
+        conn
+        |> put_flash(:info, "Comment updated successfully.")
```

Comment:
@samu975 what does `put_flash/2` do?

Reply:
put flash is for writing messages in this case the message is info type and the message is "comment updated successfully"